gfx/portraits/portrait_modifiers/50_beards_scripted_characters.txt

```diff
@@ -589,6 +589,7 @@ beards_scripted_characters = {
             modifier = {
                 add = 200
 				is_kelthuzad_trigger = yes
+                NOT = { is_race_no_gene_trigger = { RACE = creature_lich }
```

Comment:
Add to all beard and hair modifiers:
```
modifier = {
	factor = 0
	has_no_portrait_trigger = yes
}
```
Anyone can become a lich at some point I suppose and they will have hair and beard too.

Reply:
This didn't seem to work for some reason so I changed the code a little bit. Added to all hairstyles and beards in b39fff224466497201cf1050022d51a5fa8a5e4d.
```
modifier = {
	add = -1000
	has_no_portrait_trigger = yes
}
```

Reply:
Fixed in b0ebba5c4f4c3248c05cf2b32ef9f288cb3dc5cb.